TwoPaneLayout/library/src/main/java/com/microsoft/device/dualscreen/twopanelayout/TwoPaneLayout.kt

```diff
@@ -47,13 +49,17 @@ enum class TwoPaneMode {
  * they will be layout equally, with the potential padding in-between based on the
  * physical hinge between two screens.
  *
- * @param modifier The modifier to be applied to the TwoPane.
+ * @param modifier: The modifier to be applied to the TwoPaneLayout
+ * @param paneMode: The [TwoPaneMode] that determines when one or two panes are shown
+ * @param navController: The navController to use when navigating within the single pane container
+ * @param pane1: The content to show in pane 1
+ * @param pane2: The content to show in pane 2
  */
-
 @Composable
 fun TwoPaneLayout(
     modifier: Modifier = Modifier,
     paneMode: TwoPaneMode = TwoPaneMode.TwoPane,
+    navController: NavHostController,
```

Comment:
Can we have `navController: NavHostController = rememberNavController()`, instead of having the following 
```
@Composable
fun TwoPaneLayout(
    modifier: Modifier = Modifier,
    paneMode: TwoPaneMode = TwoPaneMode.TwoPane,
    pane1: @Composable TwoPaneScope.() -> Unit,
    pane2: @Composable TwoPaneScope.() -> Unit
)```

Reply:
This is up to us - if we want to just use one constructor, then it will force devs to also include the `androidx.navigation:navigation-compose` import (to resolve the `NavHostController` parameter)

Is that ok? Or would you prefer to keep them separate?